contextualized/dags/lightning_modules.py

```diff
@@ -22,21 +74,17 @@ def __init__(
         self,
         context_dim,
         x_dim,
-        num_archetypes=4,
-        use_dynamic_alpha_rho=True,
-        sample_specific_loss_params={"l1": 0.0, "alpha": 1e-1, "rho": 1e-2},
-        archetype_loss_params={"l1": 0.0, "alpha": 0.0, "rho": 0.0},
-        learning_rate=1e-3,
-        opt_step=50,
-        encoder_type="mlp",
-        encoder_kwargs={"width": 32, "layers": 2, "link_fn": identity_link},
-        init_mat=None,
+        sample_specific_params=DEFAULT_SS_PARAMS,
+        archetype_params=DEFAULT_ARCH_PARAMS,
```

Comment:
I prefer `sample_specific_loss_params` and `archetype_loss_params` since this makes it seem like separate args like `num_archetypes` would be contained in `archetype_params`.